app/utils/CheckYourAnswersHelper.scala

```diff
@@ -16,7 +16,6 @@
 
 package utils
 
-import controllers.routes
```

Comment:
Removing this will break scaffolds  from running on the first test without modifying the code

Reply:
I don't understand this comment